utils/mock-data-loader/bss/facilities/deal-9-facilities.js

```diff
@@ -47,6 +48,7 @@ module.exports = [
 		"facilityType": "Bond",
 		"ukefGuaranteeInMonths": null,
 		"facilityStage": "Issued",
+    "hasBeenIssued": true,
```

Comment:
Indentation here and in above files

Reply:
funny enough... in my editor they are aligned properly :|
I'll format the document though - that should sort it :)